testing/camunda-process-test-spring/pom.xml

```
@@ -25,6 +25,15 @@
     <license.header>com/mycila/maven/plugin/license/templates/APACHE-2.txt</license.header>
   </properties>
 
+  <profiles>
+    <profile>
+      <id>M4-JVM-BUG</id>
```

Could you rename the profile id, what about: 

```suggestion
      <id>M4-JVM</id>
```